product-updates/knowledge-center/create-new-login.html

```diff
@@ -285,7 +296,13 @@ <h2 class="usa-process-list__heading">Create your Account</h2>
           <li class="usa-process-list__item">
             <h2 class="usa-process-list__heading">Confirm your Email Address</h2>
             <p>
-              Check your email inbox for a message from Login.gov and select 'Confirm email address'. This will take you back to the Login.gov website.
+              Check your email inbox for a message from Login.gov and select 'Confirm email address'. This will take you back to the Login.gov website.<br>
+              <br>
+              If you did not receive an email, there are a few steps you can take:
+              <ul>
+                <li>Check your email’s spam or junk folder for an email from 'no-reply@tanfdata.acf.hhs.gov'.</li>
```

Comment:
@reitermb the confirm email comes from login.gov. the emails from no-reply start after a user has requested access to TDP. can you move this reminder to the following steps: request access, approval, data and data submitted. 

also are we adding screenshots of denied, deactivation warnings, and deactivated emails anywhere in the knowledge center?

Reply:
@lilybl1 @reitermb @stevenino:

- @lilybl1 will fix where no-reply is located. 
- @lilybl1 will add a sentence about denial emails in italics under approval sections. but no pictures. 
- we need a new ticket to add pictures and description of denial, inactivity, and deactivation emails later after 2.2

I will approve this today with these changes.

Reply:
lets wait until @reitermb returns, hopefully tomorrow.

Reply:
@ADPennington @stevenino @lilybl1 

Playing a little catchup here but I /think/ all the discussed changes--both from Friday and yesterday--should now be committed.